TrueSDK/Core/DislaimerView.m

```diff
@@ -0,0 +1,95 @@
+//
+//  DislaimerView.m
+//  TrueSDK
+//
+//  Created by Ashutosh Roy on 10/08/21.
+//  Copyright © 2021 True Software Scandinavia AB. All rights reserved.
+//
+
+#import "DislaimerView.h"
+#import "TCUtils.h"
+
+NSString *const kTCTermsURL = @"https://developer.truecaller.com/phone-number-verification/privacy-notice";
+
+@implementation DislaimerView
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self != nil) {
+        [self setup];
+    }
+    return self;
+}
+
+- (void)setup
+{
+    [self addDisclaimerSubViews:self];
+}
+
+
+-(void)addDisclaimerSubViews:(UIView *)view {
+    UILabel *label = [UILabel new];
```

Comment:
Remove this method and add label addition and close button addition methods to setup function